README.md

```diff
@@ -18,6 +18,13 @@ If bundler is not being used to manage dependencies, install the gem by executin
 
     $ gem install flatito
 
+### Nixpkgs package
+
+It is also available as [nixpkgs](https://github.com/ceritium/flatito/issues/8) thanks to @Rucadi
+
+    $ nix run nixpkgs#flatito
```

Comment:
Yep, this command is correct and should work.
![image](https://github.com/ceritium/flatito/assets/6445619/f7c9882f-9f64-4c23-82f1-b01f6236aa7c)